Include the endian compat header; __BYTE_ORDER is used below.
To be sent upstream.

Index: spa/plugins/audiomixer/mix-ops.h
--- spa/plugins/audiomixer/mix-ops.h.orig
+++ spa/plugins/audiomixer/mix-ops.h
@@ -3,6 +3,7 @@
 /* SPDX-License-Identifier: MIT */
 
 #include <spa/utils/defs.h>
+#include <spa/utils/endian.h>
 
 typedef struct {
 #if __BYTE_ORDER == __LITTLE_ENDIAN
